python_scripts/03_categorical_pipeline_ex_01.py

```diff
@@ -52,10 +52,10 @@
 data_categorical = data[categorical_columns]
 
 # %% [markdown]
-# We filter our dataset that it contains only categorical features.
+# We filtered our dataset so that it contains only categorical features.
```

Comment:
To be honest this line can actually be removed since the text in the previous cell says it already